test/format/checkpoint.c

```diff
@@ -142,7 +140,18 @@ checkpoint(void *arg)
         else
             trace_msg(session, "Checkpoint #%u start (%s)", ++counter, ckpt_config);
 
-        testutil_check(session->checkpoint(session, ckpt_config));
+        ret = session->checkpoint(session, ckpt_config);
+        /*
+         * Because of the concurrent activity of the sweep server, it is possible to get EBUSY when
+         * we are trying to remove an existing checkpoint as the sweep server may be interacting
+         * with a dhandle associated with the checkpoint being removed. There is a specific scenario
```

Comment:
I am not sure if the last sentence is required. I think the first one covers what is really important - there can be concurrent access. Let me know what you think.

Reply:
I think the last sentence doesn't add a lot and agree the first sentence covers the important part.

Reply:
Thanks, updated comment in f59e14e6fde2a93ccabf7ad358c163f857cf67df.